src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  background-color: $white-accent;
+  color: $main-text-color;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: solid 1px $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__description {
+    margin-bottom: 16px;
+
+  }
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $gray-color;
+    margin: 0;
+  }
+
+  &:hover .card__title {
+    color: #34568b;
+  }
+
+  &__reviews {
+    display: flex;
+    margin-bottom: 24px;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviwes-value {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
```

Comment:
Let's fix it everywhere
```suggestion
  &__reviews {
    display: flex;
    margin-bottom: 24px;
    justify-content: space-between;
    align-items: center;
    
    &-value {
      ....
    }
  }
```

Reply:
![image](https://github.com/mate-academy/layout_catalog/assets/143791750/47614fe6-bbbb-46bc-95b3-226d9ef389ff)